src/interfaces/node.cpp

```diff
@@ -53,7 +53,7 @@ class NodeImpl : public Node
     {
         return gArgs.ParseParameters(argc, argv, error);
     }
-    bool readConfigFiles(std::string& error) override { return gArgs.ReadConfigFiles(error); }
+    bool readConfigFiles(std::string& error) override { return gArgs.ReadConfigFiles(error, true); }
```

Comment:
Please use named arguments for literal primitive types. (`/* ignore_invalid_keys */ true`)